### PR TITLE
Feature/fix mini pupper config

### PR DESCRIPTION
**Pull request by CullenSUN**
- Linked to official mangdang's [mini_pupper_description](https://github.com/mangdangroboticsclub/mini_pupper_description) repo. 
- Copied some config files from mangdang's [minipupper_ros](https://github.com/mangdangroboticsclub/minipupper_ros/tree/master/mini_pupper) repo.
- Added support for ros2

### CODE REVIEW

**Review by Tiryoh — COMMENTED** [review]
Awesome!
I made a few modifications and it worked.

https://user-images.githubusercontent.com/3256629/191815060-3a60c111-3bd2-4ed1-96b7-c19dc0db2dac.mp4

The install instructions did not worked, so I made a PR to your branch.
https://github.com/CullenSUN/chvmp_robots/pull/1

File: configs/mini_pupper_config/launch/gazebo.launch.py [hunk 1]
```diff
@@ -45,11 +46,13 @@ def generate_launch_description():
...
     )
     declare_robot_name = DeclareLaunchArgument(
-        "robot_name", default_value="champ", description="Robot name"
+        "robot_name", default_value="mini_pupper", description="Robot name"
     )
     declare_lite = DeclareLaunchArgument(
         "lite", default_value="false", description="Lite"
     )
+
+    # TODO: check whether ros_control_file shall be declared here or in mini_pupper_description.urdf.xacro
```
Comment (CullenSUN) [inline configs/mini_pupper_config/launch/gazebo.launch.py:55]: hi @grassjelly do you know the answer, I didn't find anywhere using this ros_control_file

Comment (Tiryoh) [reply configs/mini_pupper_config/launch/gazebo.launch.py:55]: I guess we don't need to define it here, as it is defined below.

https://github.com/mangdangroboticsclub/mini_pupper_description/blob/270eb119e8c2943a1f05106eab30fc9036b2c1fb/urdf/mini_pupper_description.urdf.xacro#L1787

**Comment by grassjelly** [comment]
hey @CullenSUN . Could you remove the AMENT_IGNORE and uncomment all the description repos before I approve the PR. Appreciate the PR but please keep in mind that this is a common config repo and not just for mini pupper, so please be mindful of changes that disrupt other robot's configuration.

**Comment by Tiryoh** [comment]
Let me make excuses, this was a consideration so that people who wanted to use the ROS 2 version champ would not be troubled by errors while colcon build. However, the intention of not wanting to merge unnecessary changes besides updating the mini pupper's config makes sense and should be respected.
@CullenSUN Could you revert my commit ([321f4ad](https://github.com/chvmp/robots/pull/16/commits/321f4add24dc699a8e17ca2f4c8ccd07290a6294)) and update the pull request?

**Comment by CullenSUN** [comment]
hi @grassjelly got it. Will keep that in mind. Hi @grassjelly , @Tiryoh  please review again. thx

**Pull request merged**